src/server/scan.ts

```diff
@@ -119,18 +119,6 @@ process.on('message', (mess) => {
     }
   }
   if (config) {
-    if (config.gap !== undefined) {
-      gap = config.gap
-    }
-    if (config.count !== undefined) {
-      count = config.count
-    }
-    if (config.workDirPatterns !== undefined
-        && Array.isArray(config.workDirPatterns)
-        && config.workDirPatterns.length
-       ) {
-      customWorkDirPatterns = config.workDirPatterns
-    }
```

Comment:
why these lines are deleted?

Reply:
Please compare this branch against the current master. This logic was moved to `config`.

At this moment, github keeps outdated state of master and shows a conflict so please check it locally.